deeprank/learn/DataSet.py

```diff
@@ -1017,20 +876,6 @@ def backtransform_target(self, data):
         data += self.target_min
```

Comment:
Line 846 onwards will throw error/warning if we test with actual data.
If we set target_ordering= None, the program tries to assign lower_list, higher_list or NA_list. So it tries to assign order based on values in 'IRMSD', 'LRMSD', 'HADDOCK'. And when these are unidentified it assigns lower. 

Should we keep this section?

Reply:
Coos did you check this one?

Reply:
It looks like target_ordering is a class field that isn't used anywhere. It's only set in the _get_target_ordering method, but the value has no effect. We could just remove it, otherwise we need to implement an application for it ourselves.

Do we even need target_ordering if there's only one target?

Reply:
Yes, that was exactly my concern in my very first comment. 
We can remove target_ordering, as there is only one target.

Reply:
I removed the target_ordering here: https://github.com/DeepRank/deeprank/commit/01d0f9f8fdd7a93754063b496bbeb5a603917efc

Reply:
Thanks coos